blocks/init/wrapper/wrapper.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Template for the Wrapping Advance block.
+ *
+ * @since 1.0.0
+ * @package Eightshift_Boilerplate\Blocks.
+ */
+
+namespace Eightshift_Boilerplate\Blocks;
+
+// Used to add or remove wrapper.
+$has_wrapper = $attributes['hasWrapper'] ?? true;
+
+if ( $has_wrapper ) {
+
+  $id = $attributes['id'] ?? '';
+
+  $wrapper_main_class = 'wrapper';
+
+  $style_background_color = isset( $attributes['styleBackgroundColor'] ) && ! empty( $attributes['styleBackgroundColor'] ) ? "{$wrapper_main_class}__bg-color--{$attributes['styleBackgroundColor']}" : '';
```

Comment:
I think that `! empty` should suffice here, don't think you need the `isset` check.

Reply:
i will need to test this and will fix in the next PR